galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.selenium.manager/src/main/java/dev/galasa/selenium/codesnippet_01.md

```diff
@@ -8,20 +8,22 @@ public ISeleniumManager seleniumManager;
 ```
 
 The code creates an interface to the Selenium Manager which will allow the tester to provision WebPages to test against.
+</details>
 
-### Create a WebPage
+<details><summary>Create a WebPage</summary>
 
 ```
 IWebPage page = seleniumManager.allocateWebPage("https://galasa.dev/");
 ```
 
 The code creates a WebPage with a Selenium WebDriver controlling the browser. This object provides an interface for the tester to perform actions on the page to navigate around, check the page content and switch between windows.
 
-At the end of the test, the Selenium Manager automatically closes down the WebDriver which removes the WebPage.
+At the end of the test, the Selenium Manager automatically closes the WebDriver which removes the WebPage.
 
 There is no limit in Galasa on how many Selenium WebPages can be used within a single test. The only limit is the ability of the Galasa Ecosystem they are running on to support the Selenium WebDrivers not timing out.
```

Comment:
The only limit is the ability of the Galasa Ecosystem they are running on to support the number of Selenium WebDrivers ensuring that they do not time out.

Reply:
Thanks have updated.